check-for-travis.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+client = Octokit::Client.new :access_token => ENV['GITHUBTEACHER_TOKEN'] || raise("You need a GitHub Teacher access token")
+
+# Get the tree for a sha
+tree = client.tree("githubschool/on-demand-github-pages", "f1b10e926e7ace9862bf65b789204f22cccba1e0")
```

Comment:
The sha, `f1b10e926e7ace9862bf65b789204f22cccba1e0`, shouldn't be hard coded, we should use the sha that's submitted to us via the webhook.